src/components/Homepage/useValuesMarquee.ts

```diff
@@ -11,70 +12,84 @@ export type Pairing = {
 }
 
 export const useValuesMarquee = () => {
-  const { t } = useTranslation("page-index")
+  const { t, i18n } = useTranslation("page-index")
+  const locale = i18n.language
```

Comment:
Getting the locale from the `useTranslation` hook instead of `useRouter` (as we currently do in [menu](https://github.com/ethereum/ethereum-org-website/blob/dfed703ba53ca8b6c42e26d8a70a98fccce38204/src/components/Nav/Desktop/index.tsx#L22C1-L23C1)) to avoid potential re-renders and to decouple locale handling from routing